game-core/src/main/java/games/strategy/engine/framework/startup/ui/ServerSetupPanel.java

```diff
@@ -100,11 +105,24 @@ private void createLobbyWatcher() {
                           });
 
               LocalServerAvailabilityCheck.builder()
-                  .controller(watcher.getRemoteMessenger().getLobbyGameController())
+                  .connectivityCheckClient(
+                      HttpLobbyClient.newClient(
+                              URI.create(
+                                  HttpLobbyClient.PROTOCOL
+                                      + System.getProperty(LOBBY_HOST)
+                                      + ":"
+                                      + System.getProperty(LOBBY_HTTPS_PORT)),
+                              watcher.getLobbyMessenger().getApiKey())
+                          .getConnectivityCheckClient())
+                  .localPort(model.getMessenger().getLocalNode().getPort())
                   .errorHandler(errorHandler)
-                  .localNode(model.getMessenger().getLocalNode())
                   .build()
                   .run();
+
+              System.clearProperty(LOBBY_HOST);
+              System.clearProperty(LOBBY_PORT);
+              System.clearProperty(LOBBY_HTTPS_PORT);
+              System.clearProperty(TRIPLEA_NAME);
```

Comment:
Could you explain why this change was introduced?

Reply:
Turns out INode.getAddress().getHostAddress()... etc.. returns the interface in addition to IP address. Hence this warning https://github.com/triplea-game/triplea/pull/5304/files#diff-5c823a8b563c30a1fc7db84682121912R26

That can't be used to create a valid URI, and we need to create one for the HttpLobbyClient. We do have the host address available, but lobby watcher has the side effect of clearing it (so that way on the next game launch the magic system property does not cause the game to launch in a hosting mode). But, we do need lobby watcher as it creates a connection which gives us an api key. So.. the clear is removed from lobby watcher and moved to after we create the http client so that we can get the host address from props and also get the API key.